src/provisioning/api.ts

```diff
@@ -146,13 +146,13 @@ export class ProvisioningApi {
 
         const limiter = this.opts.ratelimit && ratelimiter({
             handler: (req, _res, next) => {
-                const info = (req as AugmentedRequest).ratelimit as RateLimitInfo;
-                const retryAfterMs = info?.resetTime ? info.resetTime.getTime() - Date.now() : null;
-                next(new ApiError("Too many requests", ErrCode.Ratelimited, 429, { retry_after_ms: retryAfterMs }));
```

Comment:
This *is* a departure from what we do in the Matrix spec, although I am happy to go along with what the HTTP standard is. We aren't beholden to Matrix here.